docs/assembly.rst

```diff
@@ -490,7 +490,7 @@ is performed by replacing the variable's value on the stack by the new value.
 
 .. code::
 
-    assembly {
+    {
```

Comment:
This is for consistency with the rest of the documentation as well as with Julia syntax.